src/main/java/com/google/api/generator/engine/writer/JavaWriterVisitor.java

```diff
@@ -337,6 +342,29 @@ public void visit(TryCatchStatement tryCatchStatement) {
     newline();
   }
 
+  /** =============================== COMMENT =============================== */
+  public String visit(LineComment lineComment){
```

Comment:
Same as above, why is this method returning a String?

Reply:
same as above. Thanks